src/Redux/actions.tsx

```diff
@@ -685,7 +685,13 @@ export const partialUpdateExternalResult = (id: number, params: object) => {
 
 // Notifications
 export const getNotifications = (params: object) => {
-  return fireRequest("getNotifications", [], params);
+  return fireRequest(
+    "getNotifications",
+    [],
+    params,
+    {},
+    new Date().getTime().toString()
```

Comment:
but the point of having a key is that we prevent redundant calls in the first place right?

maybe we can pass key to the action function itself? (similar to how provisional diagnosis and diagnosis works?

```suggestion
    altKey
```

Reply:
makes sense 👍